test/files/input_values.yaml

```diff
@@ -0,0 +1,7 @@
+capOperatorSubdomain: cap-op
```

Comment:
can you also add appName to the list of input values/sample and use that in all the relevant xsappname across uaa/saas/other service instances and other yaml(CAP Operator)?

Reply:
Added. It will also recommend the appName based on the project configuration. Users can either use that or override it with a custom name.